build/common.sh

```diff
@@ -864,40 +853,6 @@ function kube::release::create_docker_images_for_server() {
 
 }
 
-# This will pull and save docker images for addons which need to placed
-# on the nodes directly.
-function kube::release::write_addon_docker_images_for_server() {
-  # Create a sub-shell so that we don't pollute the outer environment
-  (
-    local addon_path
-    for addon_path in "${KUBE_ADDON_PATHS[@]}"; do
-      (
-        kube::log::status "Pulling and writing Docker image for addon: ${addon_path}"
-
-        local dest_name="${addon_path//\//\~}"
-        if [[ -z $("${DOCKER[@]}" images | awk '{print ($1":"$2)}' | grep "${addon_path}") ]]; then
-          kube::log::status "Addon image ${addon_path} does not exist, pulling it..."
-          "${DOCKER[@]}" pull "${addon_path}"
-        fi
-        "${DOCKER[@]}" save "${addon_path}" > "${1}/${dest_name}.tar"
-      ) &
-    done
-
-    if [[ ! -z "${BUILD_PYTHON_IMAGE:-}" ]]; then
```

Comment:
This is still needed on CoreOS for https://github.com/kubernetes/kubernetes/blob/75c1dcaacd1dae7a5a5081d6f4b5e85c64e7b935/cluster/saltbase/salt/kube-addons/kube-addons.sh#L28 (see #18488). It's one of the reasons I've been wanting to rewrite `kube-addons.sh`.